dashboard/assets/packages/ui/src/components/Chart/Chart.tsx

```diff
@@ -37,25 +44,25 @@ export default function Chart({ panel }) {
     return <div ref={ref} />
   }
 
-  let options = {
+  const options: Options = {
     width: width,
     height: 250,
     title: panel.title,
     cursor: { sync: { key: sync.key } },
     legend: { live: false },
-    series: plot.series,
+    series: plot.series as Series[],
```

Comment:
Perhaps we could find a way for SeriesPlot to return the correct type? `Type 'object | undefined' is not assignable to type 'Series[]'`